Symfony/CS/Tests/Fixer/Contrib/Php4ConstructorFixerTest.php

```diff
@@ -965,23 +965,6 @@ public function __construct()
         $this->makeTest($expected);
     }
 
-    public function testStatic()
-    {
-        $expected = <<<'EOF'
-<?php
-
-static class Foo
-{
-    static function Foo()
```

Comment:
this case is important. Please recover it, remove `static` for class-level, but keep it for method-level